README.md

```diff
@@ -6,7 +6,7 @@ Hazelcast is an open-source distributed in-memory data store and computation pla
 
 hazelcast-cpp-client is the official C++ library API for using the Hazelcast in-memory database platform. It requires C++11 support.  
 
-The library can be installed using package managers [Conan](Reference_Manual.md#111-conan-users) and [Vcpkg](Reference_Manual.md#112-vcpkg-users) or directly [from source code](Reference_Manual.md#113-install-from-source-code-using-cmake) using [CMake](https://cmake.org/).
+The library can be installed using package managers [Conan](https://github.com/hazelcast/hazelcast-cpp-client/blob/master/Reference_Manual.md#111-conan-users) and [Vcpkg](https://github.com/hazelcast/hazelcast-cpp-client/blob/master/Reference_Manual.md#112-vcpkg-users) or directly [from source code](https://github.com/hazelcast/hazelcast-cpp-client/blob/master/Reference_Manual.md#113-install-from-source-code-using-cmake) using [CMake](https://cmake.org/).
```

Comment:
why does the orıgınal not work?

Reply:
doesn't [this issue](https://github.com/hazelcast/hazelcast-cpp-client/issues/940) originate from this file?